configs/yolox/yolox_s_8x8_300e_coco.py

```diff
@@ -116,27 +129,27 @@
     warmup_by_epoch=True,
     warmup_ratio=1,
     warmup_iters=5,  # 5 epoch
-    num_last_epochs=15,
+    num_last_epochs=num_last_epochs,
     min_lr_ratio=0.05)
-runner = dict(type='EpochBasedRunner', max_epochs=300)
 
-resume_from = None
-interval = 10
+runner = dict(type='EpochBasedRunner', max_epochs=max_epochs)
 
 custom_hooks = [
-    dict(type='YOLOXModeSwitchHook', num_last_epochs=15, priority=48),
     dict(
-        type='SyncRandomSizeHook',
-        ratio_range=(14, 26),
-        img_scale=img_scale,
+        type='YOLOXModeSwitchHook',
+        num_last_epochs=num_last_epochs,
         priority=48),
     dict(
         type='SyncNormHook',
-        num_last_epochs=15,
+        num_last_epochs=num_last_epochs,
         interval=interval,
         priority=48),
     dict(type='ExpMomentumEMAHook', resume_from=resume_from, priority=49)
 ]
 checkpoint_config = dict(interval=interval)
-evaluation = dict(interval=interval, metric='bbox')
+evaluation = dict(
+    save_best='auto',
+    interval=interval,
+    dynamic_intervals=[(max_epochs - num_last_epochs, 1)],
```

Comment:
Setting these args at the same time is a little bit confusing here. Can we only use `dynamic_intervals` here and add some comments?

Reply:
There will be bc breaking.

Reply:
This will cause conflict